orc8r/cloud/deploy/azure/aks-pipeline.yml

```diff
@@ -30,69 +25,200 @@
 #   vals.yml
 
 parameters:
-  - name: orc8rDbPassword
+  - name: aksServiceConnection
+    type: string
+  - name: aksNamespace
+    type: string
+  - name: helmValsFilepath
     type: string
-    displayName: Orchestrator DB Password
-  - name: orc8rDbHost
+  - name: deploymentStage
+    type: string
+  - name: azureServiceConnection
+    type: string
+  - name: keyVaultName
     type: string
-    displayName: Orchestrator DB Host
 
 steps:
 - checkout: self
   path: s
-- checkout: Overrides
-  path: overrides
-- task: Bash@3
-  displayName: Setup secrets for Helm deployment
-  inputs:
-    targetType: 'inline' 
-    script: |
-      mkdir -p ~/secrets
-      cp -r $(Agent.BuildDirectory)/overrides/certs/ ~/secrets/certs/
-      ls ~/secrets/certs
 
-      mkdir -p ~/secrets/envdir
-      echo "STREAMER,SUBSCRIBERDB,METRICSD,CERTIFIER,BOOTSTRAPPER,METERINGD_RECORDS,ACCESSD,OBSIDIAN,DISPATCHER,DIRECTORYD" > ~/secrets/envdir/CONTROLLER_SERVICES
-      echo "dbname=orc8r user=orc8r password=${{ parameters.orc8rDbPassword }} host=${{ parameters.orc8rDbHost }}" > ~/secrets/envdir/DATABASE_SOURCE
+# - task: Bash@3
+#   displayName: Setup secrets for Helm deployment
```

Comment:
do these commented out steps need to stay? ( assuming i'm reading this correctly)